core/assets/bundles/bundle_in_ID.properties

```diff
@@ -1236,39 +1236,39 @@ team.green.name = hijau
-hint.placeTurret = Place \uf861 [accent]Turrets[] to defend your base from enemies.\n\nTurrets require ammo - in this case, \uf838copper.\nUse conveyors and drills to supply them.
-hint.breaking = [accent]Right-click[] and drag to break blocks.
-hint.breaking.mobile = Activate the \ue817 [accent]hammer[] in the bottom right and tap to break blocks.\n\nHold down your finger for a second and drag to break in a selection.
-hint.research = Use the \ue875 [accent]Research[] button to research new technology.
-hint.research.mobile = Use the \ue875 [accent]Research[] button in the \ue88c [accent]Menu[] to research new technology.
-hint.unitControl = Hold [accent][[L-ctrl][] and [accent]click[] to control friendly units or turrets.
-hint.unitControl.mobile = [accent][Double-tap[] to control friendly units or turrets.
-hint.launch = Once enough resources are collected, you can [accent]Launch[] by selecting nearby sectors from the \ue827 [accent]Map[] in the bottom right.
-hint.launch.mobile = Once enough resources are collected, you can [accent]Launch[] by selecting nearby sectors from the \ue827 [accent]Map[] in the \ue88c [accent]Menu[].
-hint.schematicSelect = Hold [accent][[F][] and drag to select blocks to copy and paste.\n\n[accent][[Middle Click][] to copy a single block type.
-hint.conveyorPathfind = Hold [accent][[L-Ctrl][] while dragging conveyors to automatically generate a path.
-hint.conveyorPathfind.mobile = Enable \ue844 [accent]diagonal mode[] and drag conveyors to automatically generate a path.
-hint.boost = Hold [accent][[L-Shift][] to fly over obstacles with your current unit.\n\nOnly a few ground units have boosters.
-hint.command = Press [accent][[G][] to command nearby units into formation.
-hint.command.mobile = [accent][[Double-tap][] your unit to command nearby units into formation.
-hint.payloadPickup = Press [accent][[[] to pick up small blocks or units.
-hint.payloadPickup.mobile = [accent]Tap and hold[] a small block or unit to pick it up.
-hint.payloadDrop = Press [accent]][] to drop a payload.
-hint.payloadDrop.mobile = [accent]Tap and hold[] an empty location to drop a payload there.
-hint.waveFire = [accent]Wave[] turrets with water as ammunition will automatically put out nearby fires.
-hint.generator = \uf879 [accent]Combustion Generators[] burn coal and transmit power to adjacent blocks.\n\nPower transmission range can be extended with \uf87f [accent]Power Nodes[].
+hint.desktopMove = Tekan [accent][[WASD][] untuk bergerak.
+hint.zoom = [accent]Skrol[] untuk membesarkan atau mengecilkan layar.
+hint.mine = Dekati \uf8c4 bijih tembaga dan [accent]tekan[] untuk menambangnya secara manual.
+hint.desktopShoot = [accent][[Klik kanan][] untuk menembak.
+hint.depositItems = Untuk mentransfer item, tekan item dari inti.
```

Comment:
drag from your ship to the core = tarik item dari kapalmu(?)/mech/unit kamu ke inti

Reply:
gitu aja
tarik item nanti yg newbie gk tau maksudnya

Reply:
gak sekedar newbie aja, asal mudah dimengerti semua kalangan cukup kok
gw bingungnya yang bagian "tekan item dari inti"

Reply:
coba pake punya lu
klo nanti ada yg protes coba pake punya gw